controllers/rcj_soccer_referee_supervisor/referee/progress_checker.py

```diff
@@ -0,0 +1,44 @@
+import math
+
+
+class ProgressChecker(object):
+    def __init__(self,
+                 steps: int,
+                 threshold: float):
+        self.steps = steps
+        self.threshold = threshold
+        self.prev_position = None
+        self.reset()
+
+    def reset(self):
+        self.A = [0 for _ in range(self.steps)]
```

Comment:
What is `self.A`? We should probably set a self-descriptive name or document it.

Reply:
Well, an extreme abbreviation of "Array" :)

This is good feedback though, I believe the latest version should make it a bit more explicit.